packages/web-components/src/components/card-group/__stories__/README.stories.mdx

```diff
@@ -42,15 +38,35 @@ import '@carbon/ibmdotcom-web-components/es/components/card-in-card/index.js';
       Phasellus at elit sollicitudin, sodales nulla quis, consequat libero.
     </p>
     <dds-card-cta-footer slot="footer">
```

Comment:
Can we remove the svg slot and just use `cta-type="local"`

Reply:
done! Also updated the WC & React Wrapper card-group story and readmes